### ISSUE TITLE
A11y_AzureDataStudio_Create new server group:There is no alternate text for group color in add server group page

### ISSUE THREAD

**Issue by VijayaBharathyA** [microsoft/azuredatastudio#18553]
### GitHubTags:
#A11yMAS; #A11yTCS; #A11ySev2; #DesktopApp; #BM-AzureDataStudio-Win32-Feb2022; #Azure Data Studio; #WCAG1.4.1; #Win32; #AINotInScope; #Use of color;

### Environment Details:
 Windows Version :Windows 11 Enterprise
 Application name :Azure Data studio
 
### Repro Steps:
1. Launch Azure Data Studio.
2. Select the Connections icon (if not already selected) from left bar.
3. Hover over the 'Servers' header and select the icon that has a tool tip of 'New Server Group' icon.
4. Enter the name for the Server group name and a description. And then a color for the group can also be selected from the provided color options. Then click 'OK'

### Actual Result:
There is no alternate text for group color
### Expected Result:
For selection group color there should be an alternate text 
### Attachment:
![A11y_Azuredatastudio_Create new server group](https://user-images.githubusercontent.com/100189407/155711024-ed90b394-be58-4f7b-9245-6e5c19f125a8.gif)
